moveit_ros/moveit_servo/include/moveit_servo/servo_server.h

```diff
@@ -40,49 +40,12 @@
 
 #pragma once
 
-#include <moveit_servo/servo.h>
-#include <std_srvs/srv/trigger.hpp>
+#pragma message("Header `servo_server.h` is deprecated, please use `servo_node.h`")
```

Comment:
Should we add a date for when this header and the alias will be removed, I like how drake does it https://github.com/RobotLocomotion/drake/blob/master/common/drake_deprecated.h#L59

Reply:
I don't this is necessary. We'll remove it with the next major (=API breaing) release. No need to set a date for it

Reply:
I think we should just plan on deleting all the deprecated APIs right before the next major release, Humble.  We can of course do it sooner if we think that is necessary for some other reason.  main/Rolling are not guaranteed to be API stable but we should attempt to not break it often if we can.